_contrib/source/index.html

```diff
@@ -0,0 +1,324 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <title>Have fun ❤︎</title>
+    <link rel="stylesheet" href="assets/stylesheets/application.css">
+    <script type="text/javascript" src="assets/javascripts/application.js"></script>
+  </head>
+  <body class="container">
+    <br>
+    <br>
+    <br>
+    <h1>
+      HTML5 form types
+    </h1>
+
+    <p class="card">
+      Lorem ipsum dolor sit amet, <a href="#" class="text-link">consectetur adipisicing</a> elit, sed do eiusmod
+      tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
+      quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+      consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
+      cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
+      proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+    </p>
+
+    <h2>
```

Comment:
Might be nice to wrap each portion that begins with an `h2` in its own `section` for clear division of content. Or something like all things with `button` in a `section` and all `checkbox` things in a `section`